addons/account/views/account_lock_exception_views.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="view_account_lock_exception_form" model="ir.ui.view">
+            <field name="name">account.lock_exception.form</field>
+            <field name="model">account.lock_exception</field>
+            <field name="arch" type="xml">
+                <form string="Account Lock Exception">
+                    <header>
+                        <field name="active" invisible="True"/>
+                        <field name="state" widget="statusbar" statusbar_visible="active,expired" invisible="not active"/>
```

Comment:
Then the "Revoked" shows up in the status bar of revoked exceptions.
![image](https://github.com/user-attachments/assets/fda6897b-7ffc-4679-a0c4-6a0cc29f5d75)
I think it is a bit weird since the flow is more (active -> revoked or expired)
I will check with PO too